build/packaging/suseconnect-ng.changes

```diff
@@ -1,8 +1,14 @@
 -------------------------------------------------------------------
-Tue Aug 15 15:49:20 UTC 2024 - Felix Schnizlein <fschnizlein@suse.com>
-
-- Update version to 1.11 (unreleased)
-  * Add detection for container runtimes.
+Thu Jul 10 12:03:20 UTC 2024 - Miquel Sabaté <msabate@suse.com>
+
+- Update version to 1.11
+  - Added uname as collector (8858fc1d7693, 191d782054a9 and b99d42fda31e)
```

Comment:
do we need the sha sums for the individual commits?

Reply:
Came for free with the git log command I used 😄 But if you want we can remove it, np

Reply:
I would remove them, they kinda confusing in the context of a spec file I would argue. But in the end your call!

Reply:
Removed 😉